README.md

```diff
@@ -29,11 +29,30 @@ Package | Version
 [dotnet tool](https://www.nuget.org/packages/dotnet-fsharplint/) | [![NuGet Status](http://img.shields.io/nuget/v/dotnet-fsharplint.svg?style=flat)](https://www.nuget.org/packages/dotnet-fsharplint/)
 [API](https://www.nuget.org/packages/FSharpLint.Core/) | [![NuGet Status](http://img.shields.io/nuget/v/FSharpLint.Core.svg?style=flat)](https://www.nuget.org/packages/FSharpLint.Core/)
 
-## Build Status
+## How to build application
 
-Mono 4.0.2 | .NET Core 3.1
----------- | -------------
-[![Mono CI Build Status](https://travis-ci.org/fsprojects/FSharpLint.svg?branch=master "Build Status")](https://travis-ci.org/fsprojects/FSharpLint) | [![.NET CI Build status](https://ci.appveyor.com/api/projects/status/l4d22kby012cb7jf "Build Status")](https://ci.appveyor.com/project/duckmatt/fsharplint-231)
+1. Make sure you've installed .Net Core version defined in [global.json](global.json)
+2. Run `dotnet tool restore` to install all developer tools required to build the project
+3. Run `dotnet fake build` to build default target of [build script](build.fsx)
+4. To run tests use `dotnet fake build -t Test`
+5. To build documentation use `dotnet fake build -t Docs`
+
+## How to work with documentation
+
+1. Make sure you've installed .Net Core version defined in [global.json](global.json)
+2. Run `dotnet tool restore` to install all developer tools required to build the project
+3. Run `dotnet fake build` to build default target of [build script](build.fsx)
+4. Build documentation to make sure everything is fine with `dotnet fake build -t Docs`
+5. Go to docs folder `cd docs` and start Fornax in watch mode `dotnet fornax watch`
+6. Your documentation should be now accessible on `localhost:8080` and will be regenerated on every file save
+
+## How to release
+
+1. Update [CHANGELOG.md](./CHANGELOG.md) by adding new entry (`## [X.Y.Z]`) and commit it.
+2. Create version tag (`git tag vX.Y.Z`)
+3. Run `dotnet fake build -t Pack` to create the nuget package and test/examine it locally.
+4. Push the tag to the repo `git push origin vX.Y.Z` - this will start CI process that will create GitHub release and put generated NuGet packages in it
+5. Upload generated packages into NuGet.org
```

Comment:
Can we make this part automated? Today AppVeyor pushes to NuGet. It would be ideal if 2. was create and push version tag, from there the CI can package and deploy

Reply:
Yes actually, I already updated it to do exactly that, just forgot to update the documentation.

Reply:
Ah cool 👍